src/main/resources/scripts/drop fixes.sk

```diff
@@ -2,13 +2,12 @@
 # these triggers fix some drops.
 #
 
-on break of glass:
-	drop glass
-
-on break of glass pane:
-	drop glass pane
-
-on break of bookshelf:
-	cancel event
-	set block to air
-	drop bookshelf
+on break:
+	if event-block is glass:
+		drop glass
+	else if event-block is glass pane:

```

Comment:
I wouldn't say this change is actually good. Keeping the triggers modularized isn't a bad practice.

Reply:
will revert